soh/soh/SohMenuBar.cpp

```diff
@@ -1163,6 +1163,12 @@ void DrawEnhancementsMenu() {
                 }
             }
 
+            UIWidgets::Spacer(0);
+            UIWidgets::PaddedEnhancementCheckbox("Hurt Container Mode", "gHurtContainer", true, false);
```

Comment:
```suggestion
            if (UIWidgets::PaddedEnhancementCheckbox("Hurt Container Mode", "gHurtContainer", true, false)) {
                UpdateHurtContainerModeState(!CVarGetInteger("gHurtContainer", 0));
            }
```
the `!` makes this feels a little counterintuitive but i'm pretty sure we run the code in the if block before the cvar gets updated

Reply:
Should mods.cpp functions be accessible to the SohMenuBar.cpp? I checked in VS and it natively doesn't appear to know that function exists.

Reply:
it would be something you would need to declare in the mods header and then include yea

Reply:
New Commit to address, changes made as needed and the Header updated. Also needed to remove the `!` in order for the state to update correctly. the `!` returned the opposite value expected.